admin/tools/docker/work/scripts/eups-declare.sh

```diff
@@ -60,10 +60,8 @@ if [ $# -ne 0 ] ; then
     exit 2
 fi
 
-. /qserv/scripts/params.sh
-
-. $INSTALL_DIR/loadLSST.bash
+. /qserv/stack/loadLSST.bash
```

Comment:
I understand, I was using a parameter, but in practice it never changes (we are inside a Docker container so /qserv/stack might be a good convention), so I go back to hardcoded, which avoid managing a param file between all scripts/containers and ease maintenance (if you change the param file in the release container then you have to generate all child container, which is not easy).

I'll implement a more refined mechanism if have a parameter for stack install dir become useful.